crates/dev-maker/fetch_rates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# A script to fetch historic rates from Bitmex between a specifc time window
+
+# BitMEX API base URL
+API_URL="https://www.bitmex.com/api/v1"
+
+SYMBOL="XBTUSD"
+
+# Set start and end timestamps for the date range (March 20, 2023, to March 20, 2024)
+START_TIMESTAMP="2023-03-20T00:00:00.000Z"
```

Comment:
nit: How about making the script more generic by automatically calculating the start and the end timestamp based on the current date?

Reply:
I asked chat-gtp and the `date` library on linux works differently than on mac, hence, I hardcoded it 😬